test/test_jit_cuda_fuser.py

```diff
@@ -3536,7 +3572,7 @@ def forward(self, inputs : torch.Tensor, bias : torch.Tensor):
 
         self.assertEqual(o.dtype, jit_o.dtype)
         self.assertTrue(self._compare("comparing output failed", o, jit_o, error))
-        graph = t_jit.graph_for(x)
+        graph = t_jit.graph_for(x, bias)
```

Comment:
errr... how was this test passing before the change?! bias doesn't seem to be an optional argument.

Reply:
I was surprised why the test didn't complain. 🤷🏼‍♂️